pages/index.html

```diff
@@ -4,9 +4,19 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Index</title>
+  <link href="/styles/style.css" rel="stylesheet">
+  
 </head>
+ 
 <body>
-  
+  <section class ="container">
+    <h1><em>Amanda Scott</em></h1>
+    <nav>
+    <p><a href = "http://127.0.0.1:5500/pages/about.html"><i>About Me</i></a></p>
```

Comment:
While the links in a `nav` don't awlays need to be put into a `ul` tag, try to be consistent across your pages.